ros/src/tl_detector/light_classification/tl_classifier_site.py

```diff
@@ -0,0 +1,105 @@
+import rospy
+import tensorflow as tf
+import numpy as np
+from styx_msgs.msg import TrafficLight
+
+MIN_CLASSIFICATION_CONFIDENCE = 0.85
+inference_model_path = 'models/tl_site_mobilenet_1_224_graph.pb'
+
+
+class TLClassifierSite(object):
+    def __init__(self):
+        # Load classifier
+
+        self.graph = tf.Graph()
+        graph_def = tf.GraphDef()
+
+        with open(inference_model_path, "rb") as f:
+            graph_def.ParseFromString(f.read())
+        with self.graph.as_default():
+            tf.import_graph_def(graph_def)
+
+        input_name = "import/input"
+        output_name = "import/final_result"
+
+        # Get input tensor from the graph
+        self.image_tensor = self.graph.get_operation_by_name(
+            input_name).outputs[0];
+        # Get classification tensor from the graph
+        self.classification_tensor = self.graph.get_operation_by_name(
+            output_name).outputs[0];
+
+        with self.graph.as_default():
+            self.input_tensor = tf.placeholder(tf.float32, [None, None, 3])
+            float_caster = tf.cast(self.input_tensor, tf.float32)
+            dims_expander = tf.expand_dims(float_caster, 0);
+            resized = tf.image.resize_bilinear(dims_expander, [224, 224])
+            self.image_normalized = tf.divide(tf.subtract(
+                resized, [128]), [128])
+
+    def get_classification(self, image):
+        """Determines the color of the traffic light in the image
+
+        Args:
+            image (cv::Mat): image containing the traffic light
+
+        Returns:
+            int: ID of traffic light color
+                 (specified in styx_msgs/TrafficLight)
+
+        """
+
+        with self.graph.as_default():
+            with tf.Session(graph=self.graph) as sess:
```

Comment:
Please store the session in the constructor, otherwise it creates a new one for every iteration. Can be done in the same way as in the `sim` version